docs/INSTALLATION.md

```diff
@@ -8,13 +8,20 @@
     npm install @clr/ui
     ```
 
-2.  Include clr-ui.min.css in your HTML file:
+2.  Include @cds/core/global.min.css and @cds/core/styles/theme.dark.min.css in your HTML file:
+
+    ```
+    <link rel="stylesheet" href="path/to/node_modules/@cds/core/global.min.css">
+    <link rel="stylesheet" href="path/to/node_modules/@cds/core/styles/theme.dark.min.css">
+    ```
+
+3.  Include clr-ui.min.css in your HTML file:
 
     ```
     <link rel="stylesheet" href="path/to/node_modules/@clr/ui/clr-ui.min.css">
     ```
```

Comment:
Wouldn't be better to combine SCSS into 1 point?!
Example:

```suggestion
2.  Include following files `clr-ui.min.css`, `@cds/core/global.min.css` and `@cds/core/styles/theme.dark.min.css` in your HTML file:

    ```
    <link rel="stylesheet" href="path/to/node_modules/@clr/ui/clr-ui.min.css">
    <link rel="stylesheet" href="path/to/node_modules/@cds/core/global.min.css">
    <link rel="stylesheet" href="path/to/node_modules/@cds/core/styles/theme.dark.min.css">
    ```
```

Reply:
This is much better. Thanks!